test/test-patientsController.js

```diff
@@ -655,13 +655,13 @@ describe('patientsController', function() {
       '/psychologue/modifier-patient?patientid=' + patientId)
     })
 
-    it('should refuse ine with non-aphanumeric chars', function(done) {
+    it('should refuse validation with ine length > 50 chars', function(done) {
       const patientId = '67687f5a-b9cf-4023-9258-fa72d8f1b4b3'
       shouldFailUpdatePatientInputValidation(done, {
         'patientid': patientId,
         'firstnames': 'Blou Blou',
         'lastname': 'Nom',
-        'ine': '1234567890à',
+        'ine': '515151515151515151515151551515151515151515151515151',
```

Comment:
Same as [previously](https://github.com/betagouv/sante-psy/pull/151/files#diff-d161264d99521eec5ac316a00c8830d6cc3e3c7ed8190c423620e87eb0a2a3b8R84) where using `repeat()` could be handy to understand quickly what is going on.